### PR TITLE
Ashwini_Assignment_35

### PR DESCRIPTION
**Pull request by AshwiniBalki**
### Assignment Description :notebook:
Assignment - 35 : 4th Feb'2024
Return the sum of even index [0,2,4,6]
input : {3, 6, 9, 11, 15, 20, 4, 26}
Output : 31
hint 3, 9, 15, 4

### Evidence :file_folder:
image

### Checklist before requesting a review :ballot_box_with_check:

- [x] I have performed a self-review of my code
- [x] My changes generate no compile time errors

### CODE REVIEW

File: src/ashwini_b/SumOfEvenIndex.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+hint 6, 20, 4, 26
+ */
+public class SumOfEvenIndex {
+
+	int sumOfEvenIndex(int[] num) {
+		int sum = 0;
+		for (int index = 0; index < num.length; index++) {
+			if (index % 2 == 0 || index == 0) {
+				sum = sum + num[index];
+				// System.out.println(num[index]);
```
Comment (MargiVegada) [inline src/ashwini_b/SumOfEvenIndex.java:17]: Don't push unnecessary commented code

File: src/ashwini_b/SumOfEvenIndex.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+input : {3, 6, 9, 11, 15, 20, 4, 26}
+Output : 60
+hint 6, 20, 4, 26
+ */
+public class SumOfEvenIndex {
+
+	int sumOfEvenIndex(int[] num) {
+		int sum = 0;
+		for (int index = 0; index < num.length; index++) {
+			if (index % 2 == 0 || index == 0) {
```
Comment (MargiVegada) [inline src/ashwini_b/SumOfEvenIndex.java:15]: Why you add this condition index==0

**Review by MargiVegada — CHANGES_REQUESTED** [review]
Incorrect evidence

File: src/ashwini_b/SumOfEvenIndex.java (hunk 1 above)
Comment (AshwiniBalki) [reply src/ashwini_b/SumOfEvenIndex.java]: removed comment.

File: src/ashwini_b/SumOfEvenIndex.java (hunk 2 above)
Comment (AshwiniBalki) [reply src/ashwini_b/SumOfEvenIndex.java]: removed condition.

**Pull request merged**